styleguide/public/styleguide/img/svg/accolade-inverse.svg

```diff
@@ -0,0 +1 @@
+<svg id="Laag_1" data-name="Laag 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Iconen_32px_clean</title><path d="M16,0A16,16,0,0,0,32,16V0Z"/><path d="M16,0A16,16,0,0,1,0,16V0Z"/></svg>
```

Comment:
deze file is nergens gebruikt, toch?

Reply:
Neen, voorlopig niet, maar die zal normaal gezien nog nodig zijn.